src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx

```diff
@@ -39,6 +39,31 @@ To forward your Kubernetes logs to New Relic with our plugin:
 
 <InstallFeedback />
 
+## Install Fluent Bit output plugin [#fluent-bit]
+
+New Relic has a [Fluent Bit](https://fluentbit.io/) [output plugin](https://github.com/newrelic/newrelic-fluent-bit-output) to forward your logs to New Relic log management. This plugin is also provided in a standalone Docker image that can be installed in a Kubernetes cluster in the form of a DaemonSet, also known as the Kubernetes plugin.
+
+You can install it in your cluster using our Helm chart.
+
+Despite the [newrelic-logging](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging) chart being able to work standalone, we recommend installing it as part of the [nri-bundle](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart. 
+
+The best way to install this is through our [guided installation](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/) process. This guided install can generate the Helm 3 commands required to install it ([see "Helm 3"](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/#finish-your-install)). 
+
+Alternately, you can [install it manually using Helm](/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#install-k8-helm). 
```

Comment:
@paperclypse the link in this sentence is for the guided installation, but in this section we only want to give manual instructions, right?

Reply:
Actually, I think I'm going to remove that link there. I think it distracts from the CLI commands there.